Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.Networking.sb.in

```diff
@@ -585,7 +585,9 @@
 
  ;; <rdar://problem/47598758>
 (allow mach-lookup 
-    (global-name "com.apple.nesessionmanager.content-filter"))
+    (global-name
+        "com.apple.mobileasset.autoasset"
```

Comment:
Perhaps create a separate rule for this, since the change is unrelated to <rdar://problem/47598758>?

Reply:
Good point -- I will fix that before landing.